contracts/access/TimelockController.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.9 <0.8.0;
+pragma solidity >=0.6.9 <0.9.0;
 pragma experimental ABIEncoderV2;
```

Comment:
can remove `pragma experimental ABIEncoderV2`

Reply:
I left it so that contracts compiled with solc versions older than 0.8.x still use v2 of the ABI encoder.